tests/memory_tests/README.md

```diff
@@ -0,0 +1,44 @@
+# Memory Tests
+
+This test suite contains pipelines, which are executables. 
+Memory tests measuring memory required for the use cases and fail when memory
+usage exceeds a pre-defined level.
+
+## Prerequisites
+
+To build the time tests, you need to have OpenVINO™ installed or build from source.
+
+## Measure Time
+
+To build and run the tests, open a terminal, set OpenVINO™ environment and run
+the commands below:
+
+1. Build tests:
+``` bash
+mkdir build && cd build
+cmake .. && make memory_tests
+```
+
+If you don't have OpenVINO™ installed you need to have the `build` folder, which
+is created when you configure and build OpenVINO™ from sources:
+
+``` bash
+cmake .. -DInferenceEngine_DIR=$(realpath ../../../build) && make memory_tests
```

Comment:
update README after changed CMakeLists.txt

Reply:
There should be no `InferenceEngine_DIR` option - need to set up OpenVINO correctly (install and call `setupvars` script) and after that build the tests

Refer to https://github.com/openvinotoolkit/openvino/blob/f5176f1c84b43fa89e29617e573f418983c63856/tests/time_tests/README.md